packages/js/test/plugins/auctionHouseModule/updateAuctionHouse.test.ts

```diff
@@ -115,3 +117,90 @@ test('[auctionHouseModule] it throws an error if nothing has changed when updati
   // Then we expect an error.
   await assertThrows(t, promise, /No Instructions To Send/);
 });
+
+test('[auctionHouseModule] delegate Auctioneer on Auction House update.', async (t) => {
+  // Given an existing Auction House.
+  const mx = await metaplex();
+  const auctioneerAuthority = Keypair.generate();
+
+  const { auctionHouse } = await mx
+    .auctions()
+    .createAuctionHouse({ sellerFeeBasisPoints: 200 })
+    .run();
+
+  t.false(auctionHouse.hasAuctioneer);
+
+  // When we send an update with auctioneerAuthority to delegate.
+  const { auctionHouse: updatedAuctionHouse } = await mx
+    .auctions()
+    .updateAuctionHouse(auctionHouse, {
+      auctioneerAuthority: auctioneerAuthority.publicKey,
+    })
+    .run();
+
+  // Auctioneer was delegated.
+  t.ok(updatedAuctionHouse.hasAuctioneer);
```

Comment:
Again, when the `AuctionHouse` model has more info on the auctioneer instance, we should test more assertions.

Would you mind updating the comments as well here as per my message above?